corehq/apps/translations/models.py

```diff
@@ -21,23 +22,10 @@ def set_translation(self, lang, key, value):
     def set_translations(self, lang, translations):
         self.translations[lang] = translations
 
-    
+
 class TranslationDoc(TranslationMixin):
```

Comment:
Looks like this is only used as a mixin for `StandaloneTranslationDoc`.  Can it be dropped entirely?